app/src/main/java/it/ministerodellasalute/immuni/logic/settings/CountriesOfInterestManager.kt

```diff
@@ -0,0 +1,47 @@
+package it.ministerodellasalute.immuni.logic.settings
+
+import it.ministerodellasalute.immuni.logic.exposure.models.CountryOfInterest
+import it.ministerodellasalute.immuni.logic.exposure.repositories.ExposureReportingRepository
+import it.ministerodellasalute.immuni.logic.settings.repositories.ConfigurationSettingsStoreRepository
+import org.koin.core.KoinComponent
+
+class CountriesOfInterestManager(
+    private val exposureReportingRepository: ExposureReportingRepository,
+    private val settingsRepository: ConfigurationSettingsStoreRepository
+) : KoinComponent {
+
+    fun setCountriesOfInterest(listCountries: List<CountryOfInterest>) {
+        exposureReportingRepository.setCountriesOfInterest(listCountries)
+    }
+
+    fun getCountriesSelected(): List<CountryOfInterest> {
+        return exposureReportingRepository.getCountriesOfInterest()
+    }
+
+    fun getCountries(): MutableList<CountryOfInterest> {
+        val listCountries = mutableListOf<CountryOfInterest>()
+        val countries = (settingsRepository.loadSettings()).countries
+        for (country in countries) {
+            listCountries.add(
+                CountryOfInterest(
+                    code = country.key,
+                    fullName = country.value,
+                    insertDate = null
+                )
+            )
+        }
+        return listCountries
+    }
+
+    // Questa funzione è un alternativa alla funzione getCountries per versioni da Android N in poi
```

Comment:
I'd use english as always.
Also remove unused code if not needed. If needed for some reason, let's comment in english to follow repository convention.